gr-zeromq/python/zeromq/qa_zeromq_pubsub.py

```diff
@@ -48,5 +48,28 @@ def test_001 (self):
         self.tb.wait()
         self.assertFloatTuplesAlmostEqual(sink.data(), src_data)
 
+    def test_002(self):
+        # Test getting the port returns a port we explicitly set endpoint to
+        zeromq_pub_sink = zeromq.pub_sink(gr.sizeof_float, 1, "tcp://127.0.0.1:5556", 0)
+        self.assertEqual(5556, zeromq_pub_sink.port())
```

Comment:
Of course and that's likely an issue with existing unit tests. One option is to change all of the zmq unit tests to use ephemeral ports. Do we have anything to gain/lose by never hard coding a port? Probably nothing to lose by replacing the existing test_001 with the new test_003